ztp/ztp-policy-generator/ranPolicyGenTempExamples/common-ranGen.yaml

```diff
@@ -12,38 +12,23 @@ metadata:
     mcp: "N/A"
 sourceFiles:
   # Create operators policies that will be installed in all clusters
-  - fileName: SriovSubscription
+  - fileName: SriovSubscriptionFull
     policyName: "sriov-sub-policy"
-  - fileName: SriovSubscriptionNS
-    policyName: "sriov-sub-ns-policy"
-  - fileName: SriovSubscriptionOperGroup
-    policyName: "sriov-sub-oper-policy"
-  - fileName: PtpSubscription
+  - fileName: PtpSubscriptionFull
     policyName: "ptp-sub-policy"
-  - fileName: PtpSubscriptionNS
-    policyName: "ptp-sub-ns-policy"
-  - fileName: PtpSubscriptionOperGroup
-    policyName: "ptp-sub-oper-policy"
-  - fileName: PaoSubscription
+  - fileName: PaoSubscriptionFull
     policyName: "pao-sub-policy"
-    spec:
-    # Changing the channel value will upgrade/downgrade the operator installed version.
-      channel: "4.8"
-  - fileName: PaoSubscriptionNS
-    policyName: "pao-sub-ns-policy"
-  - fileName: PaoSubscriptionOperGroup
-    policyName: "pao-sub-oper-policy"
-  - fileName: PaoSubscriptionCatalogSource
-    policyName: "pao-sub-catalog-policy"
   - fileName: ClusterLogSubscription
     policyName: "log-sub-policy"
   - fileName: MachineConfigContainerMountNS
     policyName: "worker-mc-mount-ns-policy"
-    name: "02-worker-container-mount-namespace"
-    labels:
-      machineconfiguration.openshift.io/role: worker
+    metadata:
+      name: "02-worker-container-mount-namespace"
+      labels:
+        machineconfiguration.openshift.io/role: worker
   - fileName: MachineConfigContainerMountNS
     policyName: "master-mc-mount-ns-policy"
-    name: "02-master-container-mount-namespace"
-    labels:
-      machineconfiguration.openshift.io/role: master
+    metadata:
```

Comment:
See comment above, this should be an extra manifest at install time.

Reply:
okay, lets add it to day0 manifest.